fineract-provider/src/test/java/org/apache/fineract/common/RestAssuredFixture.java

```diff
@@ -34,14 +34,12 @@
 
     private final int httpPort;
 
-    private ResponseSpecification responseSpec;
-    private RequestSpecification requestSpec;
+    private final ResponseSpecification responseSpec;
+    private final RequestSpecification requestSpec;
 
-    public RestAssuredFixture(int http) {
-
-        this.httpPort = http;
+    public RestAssuredFixture() {
+        this.httpPort = 844;
```

Comment:
@vorburger Just curious and catching up to all the brilliant development done the past few weeks. Is this suppose to be port: 8443? or something new which I dont understand?

Reply:
@awasum well spotted, yes this looks wrong. It's surprising that it still worked - I think this line can probably be removed. I could do that, but I think there is a larger question of what direction we want to take re ITs - I'll comment on the main comment instead of inline here about that.